src/main/java/org/folio/eusage/reports/api/EusageReportsApi.java

```diff
@@ -1109,21 +1109,27 @@ static String getRanges(JsonArray ar, String startProp, String endProp) {
 
   Future<Void> getUseOverTime(Vertx vertx, RoutingContext ctx) {
     String format = ctx.request().params().get("format");
+    boolean journal;
```

Comment:
For what it's worth, when I see a variable called `journal`, my immediate assumption is that it holds a reference to a journal. For a boolean that says whether we're interested in journals or not, I would be more likely to use `isJournal` or similar.

Reply:
Changed.

Reply:
Thanks!